### ISSUE TITLE
Invisible players in multiplayer

### ISSUE THREAD

**Issue by silverszi** [tsunamayo/Starship-EVO#4519]
### Short Description of the issue:
Invisible other players (not host)
### Steps to reproduce the issue:
Open a game to multiplayer
Anyone joining is invisible but can interact
### Environment
22w04a
GPU: NVIDIA GeForce GTX 1070
### Additional information:
There is someone in this seat (note the lack of light blue outline)
![image](https://user-images.githubusercontent.com/80038133/151157099-8044a6b2-b771-457f-acee-18411ebfdd51.png)

**Comment by silverszi** [comment]
[Player.log](https://github.com/tsunamayo/Starship-EVO/files/7941475/Player.log)
Nothing on my side.

**Comment by TwoNiner29** [comment]
[Player.log](https://github.com/tsunamayo/Starship-EVO/files/7941525/Player.log)
here's my log, I'm the invisible guy

**Comment by tsunamayo** [comment]
Hi, there is only two people, one host and one client? It was fine on my side...

**Comment by ProPeach** [comment]
I played a session with one host and two clients, I was a client and I could see the other client and host too 🤔

**Comment by TwoNiner29** [comment]
> Hi, there is only two people, one host and one client? It was fine on my side...

Yes it was just two of us. I was a client and the host couldn’t see me despite reloging onto the server multiple times on my end

**Comment by Womble-TC** [comment]
> Reference

I was hosting for Peach, but after about 30mins of playtime, he and the other client went invisible. It started shortly after flying hover craft around the Ring.

**Comment by wagskydive** [comment]
I tested multiplayer with a friend also about two days ago, and we had similar problems where one of us could not see the other player. I will try to reproduce to give more details, because i cant really say if it was the host or the client, because we tried different configurations... (note that we use an external app ZeroTier to make the connection on a sort of virtual LAN)
